src/main/scala/nl.knaw.dans.easy.authinfo/components/package.scala

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.authinfo
+
+package object components {
+
+  val allowedValues = RightsFor.values.map(_.toString)
```

Comment:
Indeed I don't think this helps. Objects are static, you moved the variable from one object to another.

Reply:
You are right, here it is just moving from one object to another. I cancelled this change.